core/platform/taskqueue/cloud_taskqueue_services.py

```diff
@@ -62,7 +70,7 @@ def create_http_task(
         feconf.OPPIA_PROJECT_ID, feconf.GOOGLE_APP_ENGINE_REGION, queue_name)
 
     # Construct the request body.
-    task = {
+    task: Dict[str, Any] = {
```

Comment:
Missing TODO/justification why this is valid use of `Dict[str, Any]`.

Reply:
Moved to doc

Reply:
I agree with the justification you provided via its usage in [doc](https://docs.google.com/document/d/1LEWKK2H303eHmWg1hHN2EMIlnewB6EnEyrQoReY-M6s/edit#bookmark=id.827ops6u08cn), but I can see down below the it's basically passed to CLIENT.create_task function, which belongs to google cloud library - can we use its type stubs to provide a justification of usage of Dict[str, Any] as I mentioned in previous comments? (If yes, then add a comment on top here and elsewhere too - I see 2 other places)

Reply:
Such detailed type information is not present in this library as discussed offline.

Reply:
Done.

Reply:
Looks better, thanks!